clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackOrchestrationProvider.groovy

```diff
@@ -92,4 +94,32 @@ interface OpenstackOrchestrationProvider {
    * @param stackName
    */
   List<String> getInstanceIdsForStack(String region, String stackName)
+
+  /**
+   * Get the resource associated to an instance.
+   * @param region
+   * @param stackName
+   * @param instanceId
+   * @return
+   */
```

Comment:
That does not appear to be our pattern in the Compute and Orchestration providers - leaving for a future refactoring.